.github/workflows/Harp.SoundCard.yml

```diff
@@ -0,0 +1,103 @@
+name: Harp.SoundCard
+on:
+  push:
+    # This prevents tag pushes from triggering this workflow
+    branches: ['*']
+  pull_request:
+  release:
+    types: [published]
+  workflow_dispatch:
+env:
+  DOTNET_NOLOGO: true
+  DOTNET_CLI_TELEMETRY_OPTOUT: true
+  DOTNET_GENERATE_ASPNET_CERTIFICATE: false
+  ContinuousIntegrationBuild: true
+  CiBuildVersion: ${{github.event.release.tag_name || 'api42.42.42'}}
```

Comment:
@PathogenDavid I am assuming this fallback build version is here because of the option to do the manual workflow dispatch, but wondering whether we really need this option at all?

I feel we should probably just keep CI fully automatic at this point (similar to other `bonsai-rx` repos), and if someone wants to test deployments they can always force push in their own branch.

Reply:
I guess `bonsai-rx` does have `workflow_dispatch` too but the danger here is we end up pushing major version 42 somewhere in test packages, which I would rather avoid doing.

Reply:
> I am assuming this fallback build version is here because of the option to do the manual workflow dispatch, but wondering whether we really need this option at all?

This is incorrect. This really only exists so that the build can succeed for events other than `release` (including `push`.) It is basically a borderline-useless equivalent of `bonsai-rx/configure-build` (so useless that the unstable packages can't be published to GitHub Packages, which is why this workflow doesn't do that.)

It is not meant to exist long-term, this entire workflow is a placeholder.